packages/main/src/RPA/JavaAccessBridge.py

```diff
@@ -0,0 +1,520 @@
+
+    To identify element with more than one property `and` can be used, for example:
+
+        .. code-block:: console
+
+            role:push button and name:Clear
+
+    To address element within parent element `>` can be used, for example:
+
+        .. code-block:: console
+
+            name:Find Purchase Orders > name:NumberField
+
+    Some keywords accept element as an parameter in place of locator.
+
+    ** Interacting with elements **
+
+    By default application elements are interacted with Actions supported by the element.
+    Most common example is `click` action supported by an button element.
+
+    But because application and technology support for the actions might be limited, it is also
+    possible to opt for interaction elements by their coordinates by giving keyword parameter
+    ``action=False`` if parameter is available.
+
+    ** Inspecting elements **
+
+    The Google's `Access Bridge Explorer`_ can be used for inspecting Java application elements.
+
+    .. _Access Bridge Explorer: https://github.com/google/access-bridge-explorer
+
+    **Examples**
+
+    .. code:: robotframework
+
+        *** Settings ***
+        Library   RPA.JavaAccessBridge
+        Library   Process
+
+        *** Tasks ***
+        Write text into Swing application
+            Start Process    java -jar BasicSwing.jar    shell=${TRUE}    cwd=${CURDIR}
+            Select Window    Chat Frame
+            Type Text    role:text    text for the textarea
+            Type Text    role:text    text for the input field  index=1  clear=${TRUE}
+            Click Element    role:push button and name:Send
+
+    .. code:: python
+
+        from RPA.JavaAccessBridge import JavaAccessBridge
+        import subprocess
+
+        jab = JavaAccessBridge()
+
+        subprocess.Popen(["java", "-jar", "BasicSwing.jar"], shell=True, cwd=".", close_fds=True)
+        jab.select_window("Chat Frame")
+        jab.type_text("role:text", "text for the textarea", enter=True)
+        jab.type_text("role:text", "text for the input field", index=1, clear=True)
+        jab.click_element("role:push button and name:Send")
+    """  # noqa: E501, W605
+
+    def __init__(self):
+        self.logger = logging.getLogger(__name__)
+        self.version_printed = False
+        self.jab_wrapper = None
+        self.context_info_tree = None
+        self.pumper_thread = None
+        self._initialize()
+        self.refresh_counter = 1
+        self.display_scale_factor = (
+            ctypes.windll.shcore.GetScaleFactorForDevice(0) / 100
+        )
+
+    def _initialize(self):
+        pipe = queue.Queue()
+        self.pumper_thread = threading.Thread(
+            target=self._pump_background, daemon=True, args=[pipe]
+        )
+        self.pumper_thread.start()
+        self.jab_wrapper = pipe.get()
+        if not self.jab_wrapper:
+            raise Exception("Failed to initialize Java Access Bridge Wrapper")
+        time.sleep(1)
```

Comment:
Why the sleep? It'll block a second for a each execution that imports this library

Reply:
sleep has been now removed